Spanner/tests/Snippet/SpannerClientTest.php

```diff
@@ -267,4 +267,12 @@ public function testResumeOperation()
         $res = $snippet->invoke('operation');
         $this->assertInstanceOf(LongRunningOperation::class, $res->returnVal());
     }
+
+    public function testEmulator()
+    {
+        $snippet = $this->snippetFromClass(SpannerClient::class, 1);
+        $res = $snippet->invoke('spanner');
+        $this->assertInstanceOf(SpannerClient::class, $res->returnVal());
+        $this->assertEquals('localhost:9010', getenv('SPANNER_EMULATOR_HOST'));
```

Comment:
Rather than testing if the env var is set, should this instead read the endpoint from the SpannerClient config?

Reply:
I am following how emulator support was implemented for other projects. I will try update this and the testing along with the other emulators.